core/src/main/java/org/fao/geonet/kernel/security/openidconnect/OIDCRoleProcessor.java

```diff
@@ -329,6 +330,11 @@ public List<String> getTokenRoles(Map<String, Object> attributes, String pathToR
                 Log.debug(Geonet.SECURITY, "oidc: pathToRoles - cannot find path component named: " + path);
                 return new ArrayList<>();
             }
+            if (o instanceof String) {
```

Comment:
Arrays.asList(o) ?

Reply:
I'd have like to: I tried `Arrays.asList` as a first version of my code, but the list can be altered later on in the code, and `Arrays.asList` will create an immutable one, leading to a `java.lang.UnsupportedOperationException` being fired afterwards.